about.html

```diff
@@ -23,8 +23,20 @@
         <li class="menu-highlight"><a href="#">Stoa Campaign</a></li>
       </ul>
     </navbar>
-    <!-- Menu Mobile-->
-    <navbar class="nav"><img src="images/mobile_menu.png" alt="hamburger"></navbar>
+    <nav>
+      <a class="hamburger">
+        <img src="images/mobile_menu.png" alt="hamburger">
+      </a>
+    </nav>
+    <nav>
+
+      <ul class="mobile-menu">
+        <a class="close"><img src="images/cancel.png" alt=""></a>
+        <li class="menu-link"><a href="index.html">Home</a></li>
+        <li class="menu-link"><a href="about.html">About</a></li>
+        <li class="menu-link menu-highlight"><a href="#">Stoa</a></li>
+      </ul>
+    </nav>
   </header>
 
 
```

Comment:
- [ ] Kindly make sure to add a divider after the third section headline so your page reflects the provided design
- [ ] Also kindly consider adding text on top of the two images in this section so they have a descriptive text
- [ ] The two images in the desktop view should be aligned horizontally kindly consider styling them that way  so they look more appealing and clean 

|Your page|Provided design|
|-|-|
|![Screenshot_20230318_001908](https://user-images.githubusercontent.com/25227022/226068559-8ea77729-5fab-4dcc-8969-3e78593d14c9.png)|![Screenshot_20230318_002328](https://user-images.githubusercontent.com/25227022/226068574-235c44cf-6372-438c-9065-dcad4548c0a8.png)|